### ISSUE TITLE
Possible proxy discovery typo in kestrel.py

### ISSUE THREAD

**Issue by mrmundt** [Pyomo/pyomo#1218]
I was running the help command `pyomo --help solvers` and got this error:
```
Traceback (most recent call last):
  File "/Users/mmundt/opt/anaconda3/bin/pyomo", line 11, in <module>
    load_entry_point('Pyomo', 'console_scripts', 'pyomo')()
  File "/Users/mmundt/Documents/devel-pyomo/pyomo/pyomo/scripting/pyomo_main.py", line 92, in main_console_script
    ans = main()
  File "/Users/mmundt/Documents/devel-pyomo/pyomo/pyomo/scripting/pyomo_main.py", line 82, in main
    retval = _options.func(_options)
  File "/Users/mmundt/Documents/devel-pyomo/pyomo/pyomo/scripting/driver_help.py", line 414, in help_exec
    help_solvers()
  File "/Users/mmundt/Documents/devel-pyomo/pyomo/pyomo/scripting/driver_help.py", line 331, in help_solvers
    kestrel = pyomo.neos.kestrel.kestrelAMPL()
  File "/Users/mmundt/Documents/devel-pyomo/pyomo/pyomo/neos/kestrel.py", line 113, in __init__
    self.setup_connection()
  File "/Users/mmundt/Documents/devel-pyomo/pyomo/pyomo/neos/kestrel.py", line 129, in setup_connection
    transport.set_proxy(proxy)
  File "/Users/mmundt/Documents/devel-pyomo/pyomo/pyomo/neos/kestrel.py", line 89, in set_proxy
    self.proxy = urlparse('http://'+proxy)
NameError: name 'proxy' is not defined
```

In the file /pyomo/neos/kestrel.py, line 85-89:
```
        def set_proxy(self, host):
            self.proxy = urlparse(host)
            if not self.proxy.hostname:
                # User omitted scheme from the proxy; assume http
                self.proxy = urlparse('http://'+proxy)
```
Either `self.proxy = urlparse('http://'+proxy)` needs to be changed to `+host` or the argument for `set_proxy` needs to be amended to `def set_proxy(self, proxy)` (plus `self.proxy=urlparse(proxy)`) in order to correct this error message.

## Legal Disclaimer

By contributing to this software project, I agree to the following terms and conditions for my contributions:

   1. I agree my contributions are submitted under the BSD license.
   2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

**Comment by blnicho** [comment]
Could you update your clone of Pyomo? I think we fixed this on master last week in #1206

**Comment by mrmundt** [comment]
Ah, it did indeed! Thanks!
